### PR TITLE
enable environment variables

### PR DESCRIPTION
**Pull request by chgohlke**

### CODE REVIEW

**Comment by wavemoran** [comment]
The issue I'm having with this change is now all Lambda jobs require an environment variable per, https://github.com/terraform-providers/terraform-provider-aws/issues/1110.

**Comment by chgohlke** [comment]
@wavemoran Yeah, you're right. A possible solution is to define a default environment variable (e.g. LOGGING_LEVEL or so) that is set by default. So all Lambdas have this variable (which shouldn't be a problem) and Terraform doesn't bitch. 

Another way would be to use Terraform 0.12. But this might take a while until it is ready and reasonably stable.

File: modules/lambda/main.tf [hunk 1]
```diff
@@ -9,6 +9,10 @@ resource "aws_lambda_function" "lambda" {
   timeout       = "${var.timeout}"
   memory_size   = "${var.memory_size}"
   tags          = "${var.tags}"
+
+  environment {
```
Comment (moritzzimmer) [inline modules/lambda/main.tf]: I'd prefer a solution like in the [claranet](https://github.com/claranet/terraform-aws-lambda/blob/d1d88bdc2ae3fde845fc043f2b8a26e5895dc171/lambda.tf#L32) module making environment variables optional

File: variables.tf [hunk 2]
```diff
@@ -74,3 +74,9 @@ variable "timeout" {
   description = "The amount of time your Lambda Function has to run in seconds. Defaults to 3."
   default     = 3
 }
+
+variable "environment_variables" {
```
Comment (moritzzimmer) [inline variables.tf]: I'd prefer using `environment` as the name on all occurences(see https://www.terraform.io/docs/providers/aws/r/lambda_function.html#environment)

**Comment by chgohlke** [comment]
@moritzzimmer i made the adjustments. plz review. enviroment Variables are now optional.

**Pull request merged**